request.go

```diff
@@ -356,8 +350,12 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				} else if v.Kind() == reflect.Int {
 					at = v.Int()
 				} else {
-					er = ErrInvalidTime
-					break
+					// Return error immediately to ensure a runtime panic doesn't swallow it.
+					return &ErrorObject{
```

Comment:
@thedodd cool.  What about i18n, however.  It would indicate the need for a global config to set the "time error warning message."  This seems kinda eugh.  Any thoughts on how we can support the use case that the message can be customized?